README.md

```diff
@@ -13,6 +13,10 @@ Terraform module to setup all resources needed for setting up an AWS Elasticsear
 * [`options_rest_action_multi_allow_explicit_index`]: String(optional, "true"): Sets the `rest.action.multi.allow_explicit_index` advanced option (must be string, not bool!). If you want to configure access to domain sub-resources, such as specific indices, you must set this property to "false". Setting this property to "false" prevents users from bypassing access control for sub-resources
```

Comment:
Maybe update the docs using `terraform-docs` :)

Reply:
true :(